scd4j/src/main/java/com/datamaio/scd4j/conf/Configuration.java

```diff
@@ -156,21 +160,14 @@ private synchronized String buildCurrentExecutionPath() {
 			SimpleDateFormat df = new SimpleDateFormat("yyyyMMddHHmmss");
 			
 			Path workspace = workspace();
-			Path config = getConfig();
-			String fileName;
-			if(config!=null) {
-				fileName = workspace.relativize(config).toString().replace("config/", "");
-			} else {
-				fileName = workspace.toString().replace("config/", "");
-			}
-			
+			String fileName = this.getConfig().getFileName().toString();
```

Comment:
That is ok for me. if you make sure there will be no cases getConfig() returns null.